README.md

```diff
@@ -69,7 +69,8 @@ Put all data files to be uploaded in the same folder.
 
 ### Prepare manifest
 
-A manifest is a special metadata (TSV) file that contains information about data files to be uploaded. CLI tool will use the information in a manifest to find, validate and upload data files to Data Hub. There are 3 columns that are important to CLI tool:
+A manifest is a special metadata (TSV) file that contains information about data files to be uploaded. CLI tool will use the information in a manifest to find, validate and upload data files to Data Hub. There are 4 columns that are important to CLI tool:
```

Comment:
recommending the following changes to this section:
"
A manifest is a metadata (TSV) file that contains information about the data files to be uploaded. CLI tool will use this information to find, validate, and upload data files to Data Hub. There are 4 columns that are relevant to CLI tool:
- the column containing file IDs
- the column containing file names
- the column containing file sizes
- the column containing file MD5 checksums
"